thesis-de/content-de.tex

```diff
@@ -315,21 +315,31 @@ \subsection{Datenhaltungssystem}
 \nl%
 
 \subsection{Datenvisualisierung}
-Die Weboberfläche muss ausgewählten Benutzern eine Übersicht über die
-verbrauchten oder freien Ressourcen, wie zum Beispiel CPU-Auslastung oder
-RAM-Auslastung einzelner virtueller oder physischer Maschinen bereitstellen.
-Hierbei ist wichtig, dass die Sicherheit (Authentisierung, Authentifizierung
-und Autorisierung) der Daten gewährleistet wird. Dies bedeutet, dass sowohl die
-Verbindung zwischen Datenbank und Weboberfläche gesichert sein muss, als auch
-die Weboberfläche selbst gegen Zugriff von Unbekannten. Die Weboberfläche
-greift hier mittels API-Abfragen auf die Datenbank zu. Dies ermöglicht es, den
-Zugriff nur für die Datenvisualisierung auf die Datenbank gewährleisten zu
-können. Die Weboberfläche soll Administratoren bei Neukonfiguration von
-virtuellen Maschinen unterstützen und zeigt hier die Auslastung der einzelnen
-physischen Systeme. Der Administrator kann so gezielt die einzelnen physischen
-Maschinen besser auslasten und verteilen. Ebenfalls kann die
-Datenvisualisierung eine Schnittstelle für komplexe Abfragen und Analysen
-bereitstellen. Bei Bedarf können diese auch visuell ausgegeben werden.
+Die Datenvisualisierung ist zuständig, für die Visualisierung der erhaltenen
+Daten. Dabei soll dem Benutzer eine Übrsicht über die verbrauchten oder freien
+Ressourcen, wie zum Beispiel CPU-Auslastung oder RAM-Auslastung einzelner
+virtueller oder physischer Maschinen bereitstellen. Bei der Datenvisualisierung
+werden mit unter sensibele Daten verarbeitet, weshalb hier auf die Sicherheit
+in anbetracht auf die Authentifizierung, Authentisierung und Autorisierung
+geachtet werden muss. Innerhalb der Projektmeetings wurde die Sicherheit der
```

Comment:
Projektmeetings

Tim erklärt irgendwo oben in der PDF den Vorgang der Projektmeetings, vielleicht noch den Ref da rein?